paperqa/docs.py

```diff
@@ -808,9 +808,7 @@ async def aquery(  # noqa: C901, PLR0912, PLR0915
             pre.answer_id = answer.id
             await self.llm_result_callback(pre)
             answer.add_tokens(pre)
-            answer.context = (
-                answer.context + "\n\nExtra background information:" + str(pre)
-            )
+            answer.context += f"\n\nExtra background information:\n{pre}"
```

Comment:
There is not a newline because that is how the other contexts are presented. Don't think we want to change that. 

```suggestion
            answer.context += f"\n\nExtra background information: {pre}"
```